routes/shopifyAuth.js

```diff
@@ -104,6 +112,436 @@ module.exports = function createShopifyAuthRoutes({
         console.error('🔴 Error storing shop access token', error);
         next(error);
       }
+    },
+
+    enableCookies(request, response) {
+      const HEADING = 'Enable cookies';
```

Comment:
In the [shopify_app PR](https://github.com/Shopify/shopify_app/pull/617) we use the i18n gem to provide translations for these. Do we have something similar here? If we don't already have something in this project I'm planning on leaving that for later in the interest of time.

Reply:
We don't have something like that for express.